README.md

```diff
@@ -48,13 +51,13 @@ You will need to create an append token with permissions just with write permiss
 Once the script is instantiated, you can call it by passing parameters to it. The following would be an example "page_load" that keeps track of the referrer as well as some other string (e.g. the nick for a specific landing page):
 
 ```javascript
-_tracker('pageload', document.referrer, 'landing_page_1')
+tbt('send', 'pageload', document.referrer, 'landing_page_1')
```

Comment:
why the send?

Reply:
we can choose whatever we consider, but we use 'init' for initialization, so we need another name for sending events, that's all.

Reply:
GA uses `create` and `send`

Reply:
ok, makes sense